lib/all-badge-examples.js

```diff
@@ -0,0 +1,2035 @@
+        exampleBadgeUri: 'https://img.shields.io/travis/USER/REPO/BRANCH.svg'
+      },
+      {
+        title: 'Wercker',
+        previewBadgeUri: '/wercker/ci/wercker/docs.svg',
+        exampleBadgeUri: 'https://img.shields.io/wercker/ci/wercker/docs.svg'
+      },
+      {
+        title: 'TeamCity CodeBetter',
+        previewBadgeUri: '/teamcity/codebetter/bt428.svg',
+        exampleBadgeUri: 'https://img.shields.io/teamcity/codebetter/bt428.svg'
+      },
+      {
+        title: 'TeamCity (simple build status)',
+        previewBadgeUri: '/teamcity/http/teamcity.jetbrains.com/s/bt345.svg',
+        exampleBadgeUri: 'https://img.shields.io/teamcity/http/teamcity.jetbrains.com/s/bt345.svg'
+      },
+      {
+        title: '(full build status)',
+        exampleBadgeUri: 'https://img.shields.io/teamcity/http/teamcity.jetbrains.com/e/bt345.svg',
+        keywords: [
+          'teamcity'
+        ]
+      },
+      {
+        title: 'AppVeyor',
+        previewBadgeUri: '/appveyor/ci/gruntjs/grunt.svg',
+        exampleBadgeUri: 'https://img.shields.io/appveyor/ci/gruntjs/grunt.svg'
+      },
+      {
+        title: 'AppVeyor branch',
+        previewBadgeUri: '/appveyor/ci/gruntjs/grunt/master.svg',
+        exampleBadgeUri: 'https://img.shields.io/appveyor/ci/gruntjs/grunt/master.svg'
+      },
+      {
+        title: 'AppVeyor tests',
+        previewBadgeUri: '/appveyor/tests/NZSmartie/coap-net-iu0to.svg',
+        exampleBadgeUri: 'https://img.shields.io/appveyor/tests/NZSmartie/coap-net-iu0to.svg'
+      },
+      {
+        title: 'AppVeyor tests branch',
+        previewBadgeUri: '/appveyor/tests/NZSmartie/coap-net-iu0to/master.svg',
+        exampleBadgeUri: 'https://img.shields.io/appveyor/tests/NZSmartie/coap-net-iu0to.svg'
+      },
+      {
+        title: 'Codeship',
+        previewBadgeUri: '/codeship/d6c1ddd0-16a3-0132-5f85-2e35c05e22b1.svg',
+        exampleBadgeUri: 'https://img.shields.io/codeship/d6c1ddd0-16a3-0132-5f85-2e35c05e22b1.svg'
+      },
+      {
+        title: 'Codeship',
+        previewBadgeUri: '/codeship/d6c1ddd0-16a3-0132-5f85-2e35c05e22b1/master.svg',
+        exampleBadgeUri: 'https://img.shields.io/codeship/d6c1ddd0-16a3-0132-5f85-2e35c05e22b1/master.svg'
+      },
+      {
+        title: 'Magnum CI',
+        previewBadgeUri: '/magnumci/ci/96ffb83fa700f069024921b0702e76ff.svg',
+        exampleBadgeUri: 'https://img.shields.io/magnumci/ci/96ffb83fa700f069024921b0702e76ff.svg'
+      },
+      {
+        title: 'Magnum CI',
+        previewBadgeUri: '/magnumci/ci/96ffb83fa700f069024921b0702e76ff/new-meta.svg',
+        exampleBadgeUri: 'https://img.shields.io/magnumci/ci/96ffb83fa700f069024921b0702e76ff/new-meta.svg'
+      },
+      {
+        title: 'CircleCI',
+        previewBadgeUri: '/circleci/project/github/RedSparr0w/node-csgo-parser.svg',
+        exampleBadgeUri: 'https://img.shields.io/circleci/project/github/RedSparr0w/node-csgo-parser.svg'
```

Comment:
For a lot of these, `previewBadgeUri` and `exampleBadgeUri` are identical. Could we infer one of them if the other one is set?

Reply:
Yea, that's a great idea. I'm happy to go through and make that change before this is merged.

I'm going to stall for the moment because `try.html` keeps changing in master so there is a lot of moving data. I'm hoping my next pass through will be the last. 😄